### ISSUE TITLE
Burn cell fails to compile with USE_CXX_EOS=FALSE

### ISSUE THREAD

**Issue by ChrisDeGrendele** [starkiller-astro/Microphysics#376]
To reproduce compile `unit_test/burn_cell` with `make -j USE_CXX_EOS=FALSE` 

it gives a long list of errors related to the c++ namespace of helmholtz. It seems like the problem is not actually set up for a fortran eos?
```
/home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:153: undefined reference to `helmholtz::tlo'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:153: undefined reference to `helmholtz::tstpi'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:154: undefined reference to `helmholtz::jtmax'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:155: undefined reference to `helmholtz::dlo'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:155: undefined reference to `helmholtz::dstpi'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:156: undefined reference to `helmholtz::itmax'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:161: undefined reference to `helmholtz::f'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:161: undefined reference to `helmholtz::f'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../EOS/helmholtz/actual_eos.H:161: undefined reference to `helmholtz::f'
...
```

and 

```
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../rates/aprox_rates.H:53: undefined reference to `rv'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../rates/aprox_rates.H:53: undefined reference to `rv'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../rates/aprox_rates.H:53: undefined reference to `rv'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../rates/aprox_rates.H:53: undefined reference to `rfdm'
/usr/bin/ld: /home/cdegrendele/codes/Microphysics/unit_test/burn_cell/../../rates/aprox_rates.H:54: undefined reference to `rfd0'
...
```

**Comment by zingale** [comment]
is there a reason you want to do that?  Even with `USE_CXX_EOS=TRUE`, the Fortran interface to the EOS is still available.

**Comment by ChrisDeGrendele** [comment]
It works with `USE_CXX_REACTIONS=FALSE` which is fine for my needs, I just wasn't sure if this was expected behavior, I spent some time but I couldn't figure out why it was trying to grab the c++ version of EOS.

**Comment by dwillcox** [comment]
closing because this was fixed with #378
